### DISCUSSION TITLE
Dreambooth now works on SD 2.1 models!

### DISCUSSION THREAD

**Discussion by Hellisotherpeople** [d8ahazard/sd_dreambooth_extension#849] [General]
I have tried to train SD 2.1 for awhile now using this extension, and I was curious to know if it was finally supported, I can report that it is fully working using the same settings that I used for SD 1.5. 

I've got 3 concepts in and the results are great!

**Comment by NoteToSelfFindGoodNickname** [comment]
Which model are you using as a source?

**Reply by Hellisotherpeople** [reply]
SD 2.1

**Comment by FranciscoFornell** [comment]
Hi! How much VRAM do you have? Any special configuration? For me it's just impossible to train 2.1 768 version without OOM errors on a 3080ti 12GB, and 1.5 training was working until I updated yesterday... Now it doesn't return any errors, but it just doesn't learn anything. It keeps training until the end but the resulting model just return random images with the instance prompt. All with the parameters provided by the wizards, and using classification images and filewords to train my face, for example.
I had to revert to a previous commit to at least be able to train 1.5 again. :(
